backend-lambda/services/productServiceHelpers

```diff
@@ -0,0 +1,18 @@
+const clientService = async (pool, query, params) => {
+  let client;
+  try {
+    client = await pool.connect();
+    const result = await client.query(query, params);
+    console.log("RESULT", result);
+    return result.rows != null ? result.rows : result;
```

Comment:
This seems a bit rigid. What if we just returned result and let whatever calls this determine if it wants to unwrap it or not.

Reply:
@Andrewdt97 You're right - I was getting an unresolved promise error and needed to just make the function await a response. Committed those changes for you to review.